validator/endpoints/tasks.py

```diff
@@ -31,122 +30,62 @@
 logger = get_logger(__name__)
 
 
+TASKS_CREATE_ENDPOINT = "/v1/tasks/create"
+GET_TASKS_BY_ACCOUNT_ENDPOINT = "/v1/tasks/account/{account_id}"
```

Comment:
don't shit on your beaut of a refactor, but feels weird these are here and not in a constants somewhere.. just me